packages/api/lambdas/publish-reports.js

```diff
@@ -71,15 +71,16 @@ function publishPdrSnsMessage(
  * Publish execution record to SNS topic.
  *
  * @param {Object} eventMessage - Workflow execution message
+ * @param {string} executionArn - Step function execution ARN
  * @returns {Promise}
  */
-async function handleExecutionMessage(eventMessage) {
+async function handleExecutionMessage(eventMessage, executionArn) {
```

Comment:
Since the `executionArn` is contained in `eventMessage`, we're passing the same information into this function twice. Calling `getMessageExecutionArn(eventMessage)` in this function would remove that redundancy. The same applies to the `handleGranuleMessages` function on line 134.